pkg/resources/api_integration_test.go

```diff
@@ -24,15 +24,15 @@ func TestAPIIntegrationCreate(t *testing.T) {
 	in := map[string]interface{}{
 		"name":                 "test_api_integration",
 		"api_allowed_prefixes": []interface{}{"https://123456.execute-api.us-west-2.amazonaws.com/prod/"},
-		"api_provider":         "aws_api_gateway",
+		"api_provider":         "aws_gov_api_gateway",
 		"api_aws_role_arn":     "arn:aws:iam::000000000001:/role/test",
 	}
 	d := schema.TestResourceDataRaw(t, resources.APIIntegration().Schema, in)
 	r.NotNil(d)
 
 	WithMockDb(t, func(db *sql.DB, mock sqlmock.Sqlmock) {
 		mock.ExpectExec(
-			`^CREATE API INTEGRATION "test_api_integration" API_PROVIDER=aws_api_gateway API_AWS_ROLE_ARN='arn:aws:iam::000000000001:/role/test' API_ALLOWED_PREFIXES=\('https://123456.execute-api.us-west-2.amazonaws.com/prod/'\) ENABLED=true$`,
+			`^CREATE API INTEGRATION "test_api_integration" API_PROVIDER=aws_gov_api_gateway API_AWS_ROLE_ARN='arn:aws:iam::000000000001:/role/test' API_ALLOWED_PREFIXES=\('https://123456.execute-api.us-west-2.amazonaws.com/prod/'\) ENABLED=true$`,
```

Comment:
Should we create a new test rather than overwrite an existing test?

Reply:
Yes sure, I will create new test.